idd/Energy+.idd.in

```diff
@@ -57223,17 +57223,41 @@ Chiller:Electric:ReformulatedEIR,
        \note curve = a + b*CWS + c*CWS**2 + d*LCT + e*LCT**2 + f*CWS*LCT
        \note CWS = supply (leaving) chilled water temperature(C)
        \note LCT = leaving condenser fluid temperature(C)
-  A4 , \field Electric Input to Cooling Output Ratio Function of Part Load Ratio Curve Name
+  A4, \field Electric Input to Cooling Output Ratio Function of Part Load Ratio Curve Type
+       \note Two part-load curves are available. 
+       \note The previous one uses the leaving condenser water temperature.
```

Comment:
@rongpengzhang Please change these notes to simply state there are two options, and then defined them, using the keyword and then what it means.  No need to talk about previous and new.  Something like:
\note Two curve types are available:
\note LeavingCondenserWaterTemperature - I'll just send a suggested IDD object by direct e-mail.